lib/engines/content_block_manager/app/components/content_block_manager/content_block_edition/details/fields/enum_component.rb

```diff
@@ -7,14 +7,26 @@ def initialize(enum:, default: "", **args)
 
 private
 
+  attr_reader :enum
+
   def options
-    [blank_option, @enum].flatten.compact.map do |item|
+    options = [
       {
+        text: blank_option,
+        value: "",
+        selected: selected?(blank_option),
+      }
+    ]
+
+    enum.each do |item|
+      options.push({
         text: item,
-        value: item,
+      value: item,
         selected: selected?(item),
-      }
+      })
     end
+
+    options
```

Comment:
do you think it's worth using the EnumComponent unit test to illustrate the behaviour here? I'm not clear what's going on...

Reply:
I've made the `options` method public and tested it directly now, so hopefully that will simplify it a bit 👍